-------------------------------------------------------------------------------
Test set: com.pilalab.back.PilalabApplicationTests
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 3.737 s <<< FAILURE! - in com.pilalab.back.PilalabApplicationTests
contextLoads  Time elapsed: 0.002 s  <<< ERROR!
java.lang.IllegalStateException: Failed to load ApplicationContext
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'mainRepository' defined in com.pilalab.back.service.MainRepository defined in @EnableJpaRepositories declared on PilalabApplication: Invocation of init method failed; nested exception is java.lang.IllegalArgumentException: Not a managed type: class java.lang.Object
Caused by: java.lang.IllegalArgumentException: Not a managed type: class java.lang.Object

